packages/admin-ui/server/getWebpackConfig.js

```diff
@@ -18,7 +18,15 @@ module.exports = function({ adminMeta, entry }) {
   const rules = [
     {
       test: /\.js$/,
-      exclude: [/node_modules(?!\/@keystone-alpha\/admin-ui)/],
+      exclude: [
+        /node_modules(?!\/@keystone-alpha\/admin-ui)/,
+        // this only affects things while developing in the monorepo
+        // we do this so we can use less memory on heroku, this uses less memory
+        // since it doesn't have to compile these things with babel
+        // note that the preconstruct aliases are also disabled on heroku to enable this
+        // when we have static builds for the admin ui, we can remove this
+        ...(process.env.HEROKU === 'true' ? [/@keystone-alpha\/field/, /@arch-ui/] : []),
```

Comment:
Nice. I'd suggest a slight improvement by creating a function like

```js
isHerokuEnv = () => process.env.HEROKU
```

Makes the encapsulation and intent slightly clearer

Reply:
I extracted out `process.env.HEROKU === 'true'` into a constant `isHerokuEnv`, is that what you want? or is there a reason it should be function?